src/Features/CSharp/Portable/DocumentationComments/DocCommentConverter.cs

```diff
@@ -74,26 +70,24 @@ public override SyntaxNode Visit(SyntaxNode node)
             return node;
         }
 
-        private IEnumerable<SyntaxTrivia> ConvertDocCommentToRegularComment(DocumentationCommentTriviaSyntax structuredTrivia)
+        private void ConvertDocCommentToRegularComment(
```

Comment:
Just curious, but why do we change DocComments to RegularComments?

Reply:
> Just curious, but why do we change DocComments to RegularComments?

Primarily because doc comments are terrible to read in code.  They're desigend to be structured, and then rendered nicely by some presenter (i.e. quick-info, or html docs), but they themselves are really unpleasant to look at as the first class source of info.

We convert to just a really simple textual view that puts the content forward and basically hides all the structure tags.

Reply:
> ... hides all the structure tags ...

Also loses semantic references and syntax highlighting 😢

Reply:
just have to write some tools for that :)